### DISCUSSION TITLE
Understand parrallel execution of task by conductor server

### DISCUSSION THREAD

**Discussion by vidhi-pavan-agrawal** [Netflix/conductor#2571] [Q&A] [answered]
I have made a Dynamic workflow where have I'm 50 HTTP tasks under forktask followed by a Fork_Join task where I'm combining all the 50 Tasks. I have installed Conductor version 2.32.0. and the default installation using docker-compose ("queues.dynomite.threads": "10"). As far as my understanding goes fork task will put all tasks in a scheduled state and then HTTP tasks being system tasks will be executed by the conductor server.
if we are willing to execute tasks parallelly where I should get a response time of the workflow = time taken by the longest task (approximately), How many HTTP tasks I can run parallelly with my configuration?

**Comment by vidhi-pavan-agrawal** [comment, marked as answer]
Also, how do I increase the capacity of the Conductor server to execute more number of HTTP Tasks in parallel?

**Reply by jxu-nflx** [reply]
Hello @vidhi-pavan-agrawal you can add this property to your config: `conductor.app.system-task-max-poll-count`. The default value is 1

**Reply by aravindanr** [reply]
You can also horizontally scale by adding more servers.
